lib/request.js

```diff
@@ -59,4 +59,4 @@ function request (configVersion, globalOptions, options) {
   return got(options.path, gotOptions)
 }
 
-module.exports = {request}
```

Comment:
This didn't cause breaking changes?

Reply:
The only other necessary change was a line in `lib/trader.js`: b9b9634105835a2803a15872feec9646178a787a